src/frontend/screens/Quests/components/QuestDetailsViewPlay/index.tsx

```diff
@@ -146,13 +146,13 @@ export function QuestDetailsViewPlayWrapper({
       }}
       classNames={{ root: styles.questDetailsRoot }}
       isQuestsPage={true}
-      onPlayClick={async () =>
-        window.api.launch({
-          appName: questMeta.project_id,
-          launchArguments: '',
-          runner: 'hyperplay'
+      onPlayClick={async () => {
+        const gameId = questMeta.project_id
+        const gameInfo = await getGameInfo(gameId, 'hyperplay')
```

Comment:
I think this is fine as a workaround for now but for some people this will cause a few seconds delay, maybe more.
Ideally we should be able to get the gameinfo when the component mount and then use it from there because would make the navigation instantly.